src/index.ts

```diff
@@ -0,0 +1,121 @@
+    })
+
+    let counter = 0
+    const feeds = new Map();
+    feeds.set("Geral", feed);
+    
+    for await (const acordao of p){
+        counter++
+        
+        let [dd,mm,yyyy] = acordao.Data?.split("/") || "01/01/1900".split("/")
+        let data = new Date(parseInt(yyyy),parseInt(mm) - 1,parseInt(dd),12)
+        let id = acordao.ECLI?.startsWith("ECLI:PT:STJ:") ? `/ecli/${acordao.ECLI}` : `/${encodeURIComponent(acordao["Número de Processo"]!)}/${acordao.UUID}`
+        
+        const descritoresArray = (String) (acordao.Descritores?.Show).split(",");
+        const descritoresFormatados = descritoresArray.join(" / ");
+        const meioProcessualArray = (String) (acordao["Meio Processual"]?.Show).split(",");
+        let meioProcessualFormatado;
+        
+        if (meioProcessualArray.length > 1) {
+            meioProcessualFormatado = meioProcessualArray.join("/");
+        } 
+        else {
+            meioProcessualFormatado = acordao["Meio Processual"]?.Show;
+        }
+        
+        // Adiciona para RSS geral
+        feed.addItem({
+            title: acordao["Número de Processo"] || "Número de Processo não encontrado",
+            id: id,
+            link: publicLink + id,
+            content: acordao.Área?.Show + " - " + meioProcessualFormatado + " - " + acordao["Relator Nome Profissional"]?.Show + " - " + acordao.Secção?.Show + "<br>" +
+                    "Votação: " + acordao.Votação?.Show +  "&nbsp; &nbsp; &nbsp;" + "Decisão: " + acordao.Decisão?.Show + "<br>" +
+                    "Descritores: " + descritoresFormatados + "<br> <br>" + 
+                    "Sumário: " + acordao.Sumário || "Sumário não encontrado",
+            date: data 
+        });
+
+        if(!isAreaInFeed(feeds, acordao.Área?.Show[0])){
+            const newFeed = new Feed({
+                title: 'RSS Jurisprudência - ' + acordao.Área?.Show,
+                id: publicLink,
+                link: publicLink,
+                description: 'Latest updates from Your Website',
+                copyright: 'Supremo Tribunal da Justiça, 2024'
+            });
+            feeds.set(acordao.Área?.Show[0], newFeed)
+        }
+
+        // Adiciona para RSS da sua área
+        feeds.get(acordao.Área?.Show[0]).addItem({
+            title: acordao["Número de Processo"] || "Número de Processo não encontrado",
+            id: id,
+            link: publicLink + id,
+            content: acordao.Área?.Show + " - " + meioProcessualFormatado + " - " + acordao["Relator Nome Profissional"]?.Show + " - " + acordao.Secção?.Show + "<br>" +
+                    "Votação: " + acordao.Votação?.Show +  "&nbsp; &nbsp; &nbsp;" + "Decisão: " + acordao.Decisão?.Show + "<br>" +
+                    "Descritores: " + descritoresFormatados + "<br> <br>" + 
+                    "Sumário: " + acordao.Sumário || "Sumário não encontrado",
+            date: data 
+        });
+
+        
+        if(counter >= parseInt(process.env.RSS_MAX_FEED_SIZE!)){
```

Comment:
As discussed in chat please review this logic. We should maybe do `if( feeds.get(...).items.length <= RSS_MAX_...SIZE ) feeds.get(...).addItem` before each addItem (specific or geral). No need to break from this for loop ever.